vue-frontend/src/components/contact/ThankyouBenefitSection.vue

```diff
@@ -0,0 +1,222 @@
+<template>
+  <section class="tw-relative tw-mt-[20px] 2xl:tw-mt-[60px] tw-mb-[150px]">
+    <h1
+      class="tw-mt-8 tw-mb-8 v2-canopas-gradient-text tw-text-center tw-text-[2.5rem] md:tw-text-[3.5rem] lg:tw-text-[5rem] tw-leading-[3rem] md:tw-leading-[4rem] lg:tw-leading-[5.5rem] tw-font-inter-bold"
+    >
+      WHY US?
+    </h1>
+    <p
+      class="tw-container tw-font-inter-semibold tw-text-black-core/[0.87] tw-text-center tw-text-[1.875rem] md:tw-text-[2.813rem] lg:tw-text-[3.438rem] tw-leading-[2.438rem] md:tw-leading-[4.063rem] lg:tw-leading-[5.156rem]"
+    >
```

Comment:
Make font-size 48 px in larger devices and accordingly in mobile devices.
Currently fonts are too large